sdk/src/main/scala/com/horizen/account/state/AccountState.scala

```diff
@@ -68,22 +69,18 @@ class AccountState(val params: NetworkParams,
       // Validate top quality certificate in the end of the submission window:
       // Reject block if it refers to the chain that conflicts with the top quality certificate content
       // Mark sidechain as ceased in case there is no certificate appeared within the submission window.
-      val currentWithdrawalEpochInfo = stateView.getWithdrawalEpochInfo
-      val modWithdrawalEpochInfo: WithdrawalEpochInfo = WithdrawalEpochUtils.getWithdrawalEpochInfo(mod, currentWithdrawalEpochInfo, params)
+      val parentWithdrawalEpochInfo = stateMetadataStorage.getWithdrawalEpochInfo
+      val modWithdrawalEpochInfo = WithdrawalEpochUtils.getWithdrawalEpochInfo(mod, parentWithdrawalEpochInfo, params)
 
       // If SC block has reached the certificate submission window end -> check the top quality certificate
       // Note: even if mod contains multiple McBlockRefData entries, we are sure they belongs to the same withdrawal epoch.
-      if (WithdrawalEpochUtils.hasReachedCertificateSubmissionWindowEnd(mod, currentWithdrawalEpochInfo, params)) {
+      if (WithdrawalEpochUtils.hasReachedCertificateSubmissionWindowEnd(mod, parentWithdrawalEpochInfo, params)) {
```

Comment:
we need to use `currentWithdrawalEpochInfo` not `parentWithdrawalEpochInfo` as it was before.